docs/reference/index.md

```diff
@@ -1,17 +1,101 @@
 # Reference
 
+These guides provide technical information about Pebble.
+
+
+## Layers
+
+Your service configuration is defined as a stack of "layers".
```

Comment:
It's not just "service" configuration but also health checks and log targets. Also, "Your" feels a bit funny here. So how about:
```suggestion
Pebble configuration is defined as a stack of "layers".
```